2024/articles/2024-12-25.pod

```diff
@@ -0,0 +1,286 @@
+the CPAN has been adopted everwhere, with CI/CD systems validating any serious
+software dependency on any commit.
+
+These aspects of Perl - along with countless other things - means that twenty
+five years on not only can I still enjoy writing Perl code in more powerful and
+fun ways than I ever could, but when I need to use another language better
+suited to a particular niche, a lot of the things I loved about Perl twenty
+five years ago are already there waiting for me too.
+
+=head2 The Advent Calendar Itself
+
+Speaking of things that other languages have adopted...
+
+The story of the Perl Advent Calendar starts, like all good stories, with a
+bad idea in the pub the night before.  I don't eat chocolate, so I was lamenting
+at a Perl Mongers social meeting that there was no good advent calendars for
+me - just the ones with a little picture behind the windows.  I was saying that
+you should be able to get some other kind of reward.  And then I thought how
+about some cool code?
+
+So the Perl Advent Calendar was born.  Or it was the very next day, when I
+put the whole thing together during my lunch break.  Back then I just named a
+module for the day and included the module's POD as the advent calendar - next
+year I introduced a small description of why I'd picked it along with the pid,
+and the year after is when the full article format was introduced with links
+to search.cpan.org (and later metacpan) for the documentation.
+
+This turned out to be surprisingly popular, with the module being featured in
+NTK and on Slashdot (getting mentioned by Slashdot was a big deal in 1999).
+
+It wasn't long before there were imitators for other programming languages.  It
+became a tradition in the software world to have an Advent Calendar for your
+language of choice (or area of programming language - Perl has seen quite a few
+separate calendars for things like Perl 6, Catalyst, and even futures and
+promises).  We now even have the Advent of Code which is a small advent
+calendar of programming puzzles that's hugely popular (I, for my part, have
+never had time to do it since I was always spending time tweaking the Perl
+Avent Calendar at this time of year).
+
+Through the years the Advent Calendar became too much for me to manage on my
+own.  I could no longer keep writing all twenty five articles myself, and so
+I started taking submissions.  I eventually gave the calendar away for others
+to maintain, took it back again years later, and then finally gave it
+away again.  Like any good project, it's totally grown beyond me and has a life
+of its own.
+
+The crucial thing about the calendar - more important than any one article,
+any one year, or event any one programming language - is the idea that we can
+all enjoy learning about our favorite programming language.  Things don't have
+to be taught in boring instructions, but the instead having something to learn
+about can be silly, daft, fun, and a treat!  Why not have a story about how
```

Comment:
Doesn't read well. Maybe something like "but the instead having something to learn about" => "but instead learning about something"?